metrics/metrics.go

```diff
@@ -0,0 +1,119 @@
+package metrics
+
+import (
+	"time"
+
+	"github.com/openfresh/plasma/config"
+	"github.com/openfresh/plasma/metrics/sender"
+	metrics "github.com/rcrowley/go-metrics"
+)
+
+type Metrics struct {
+	config config.Metrics
+	sender sender.MetricsSender
+
+	GcLast metrics.Gauge
+	GcNext metrics.Gauge
+	GcNum  metrics.Gauge
+
+	GcPausePerSecond metrics.Gauge
+	GoroutineNum     metrics.Gauge
+
+	HeapAlloc   metrics.Gauge
+	HeapIdle    metrics.Gauge
+	HeapInuse   metrics.Gauge
+	HeapObjects metrics.Gauge
+	HeapSys     metrics.Gauge
+
+	MemoryAlloc   metrics.Gauge
+	MemoryFrees   metrics.Gauge
+	MemoryLookups metrics.Gauge
+	MemoryMallocs metrics.Gauge
+	MemorySys     metrics.Gauge
+
+	StackInUse metrics.Gauge
+
+	Connections metrics.Gauge
+}
+
+func NewMetrics(config config.Config) (*Metrics, error) {
```

Comment:
__[golint]__
* [golint] exported function NewMetrics should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->